### PR TITLE
Incorporate Lilly articles

### PR DESCRIPTION
**Pull request by mattsecrest**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +35,4 @@ coverage.*
 /Meta/
 inst/doc
 node_modules
+vignettes/original/data_with_weights.csv
```
Comment (gravesti) [inline .gitignore:38]:
```suggestion
vignettes/original/data_with_weights.csv

```

File: vignettes/references.bib [hunk 2]
```diff
@@ -38,4 +38,176 @@ @article{viele2014use
...
+  pages={2020},
+  year={2020}
+}
+
+@phdthesis{khanal2022semiparametric,
+  title={Semiparametric Regression Models for Clustered Right-Censored Data Under Clinical Trials and Observational Studies},
+  author={Khanal, Manoj},
+  year={2022},
+  school={The Medical College of Wisconsin}
 }
```
Comment (gravesti) [inline vignettes/references.bib:213]:
```suggestion
}

```

**Review by gravesti — COMMENTED** [review]
Looks alright, I just suggested some missing final new lines. Should we add anyone to the DESCRIPTION file? maybe with `role = "ctb"`

**Comment by mattsecrest** [comment]
@gravesti sent back at you for review!

**Review by gravesti — APPROVED** [review]

**Pull request merged**